content/implementations.md

```diff
@@ -0,0 +1,94 @@
+
+<section class="doc-note-box act-sticky">
+  <p>This page is <strong>under development</strong> and has not been approved by the working group.</p>
+</section>
+
+{::nomarkdown} {% include box.html type="start" title="Summary" %} {:/}
+The tables on these pages show how many ACT Rules different accessibility test tools and methodologies have consistently implemented. Each implementation links to a report with more details on how rules are implemented.
+{::nomarkdown} {% include box.html type="end" %} {:/}
+
+## Test Methodologies
+
+Test methodologies provide step by step instructions on how to test accessibility standards.
+See [understanding ACT consistency](#understandingactconsistency).
+
+{% assign methodologies = implementations | where: "type", "Test methodology" %}
+{% include_relative _implementations-table.html implementations=methodologies %}
+
+## Semi-automated Test Tools
+
+Tools that combine user input and automated testing to test accessibility standards.
+See [understanding ACT consistency](#understandingactconsistency).
+
+{% assign semiAutoTools = implementations | where: "type", "Semi-automated tool" %}
+{% include_relative _implementations-table.html implementations=semiAutoTools %}
+
+## Automated Test Tools
+
+Tools that fully automatically test accessibility standards.
+See [understanding ACT consistency](#understandingactconsistency).
+
+{% assign automatedTools = implementations | where: "type", "Automated tool" %}
+{% include_relative _implementations-table.html implementations=automatedTools %}
+
+## Understanding ACT Consistency
+
+Each ACT rule has examples. Vendors of test tools and methodologies run their implementations against the examples and report the outcome in a standard format. Reports are based on publicly available data.
+
+A test tool or methodology is considered **consistent** when all the following are true:
+
+- All success criteria that should fail are reported as such, and
+- all examples have the expected outcome. (I.e. a failed example should not be passed or vise versa.), and
+- there are outcomes for all examples.
+
+Automated implementations may report a "cannot tell" on some, but not all examples and still be considered consistent.
+
+A test tool or methodology is **partially consistent** when it meets some, but not all requirements. A partially consistent implementation can not include false positives; i.e. it failing any passed or inapplicable examples.
+
+Some ACT rules are "proposed". These rules are agreed on by the ACT Rules Task Force, and are awaiting implementation and approval by the Accessibility Guidelines Working Group. For details, see [about ACT Rules](../rules/about/).
+
+## Add a Tool or Methodology
+
+For information on how to add your accessibility test tool or methodology to the list of ACT implementations, visit the [ACT-Rules community group website](https://act-rules.github.io/pages/implementations/reporting/).
+
+## Disclaimer & Feedback
```

Comment:
Why are we having the disclaimer and the feedback bits all on one single heading? I think they would be more clearly understood and interpreted if they were each on their separate header.